wrappers/mason

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+# if there is a CHPL_HOME, use that to find mason
+# if that fails, fall back to the mason in path
+
+mason="none"
+
+if [ -n "$CHPL_HOME" ]; then
+  chpl_python=$($CHPL_HOME/util/config/find-python.sh 2> /dev/null)
+  exit_code=$?
+  if [ $exit_code -eq 0 ]; then
+    chpl_bin_subdir=$($chpl_python $CHPL_HOME/util/chplenv/chpl_bin_subdir.py)
+    mason="$CHPL_HOME/bin/$chpl_bin_subdir/mason"
```

Comment:
We shouod guard this with a check to make sure we find mason binary in that path.

Reply:
That is done later on

```
if [ ! -x "$mason" ]; then
  echo "Could not find suitable install of 'mason' in \$CHPL_HOME or \$PATH" >&2
  exit 1
fi
```